src/reconcile/rec_write.c

```diff
@@ -1476,9 +1477,11 @@ __rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins,
 	if (F_ISSET(r, WT_REC_LOOKASIDE) && r->las_skew_newest) {
 		if (WT_TXNID_LT(r->unstable_txn, first_upd->txnid))
 			r->unstable_txn = first_upd->txnid;
-		if (first_ts_upd != NULL &&
-		    r->unstable_timestamp < first_ts_upd->start_ts)
-			r->unstable_timestamp = first_ts_upd->start_ts;
+		if (first_ts_upd != NULL)
+			if (r->unstable_durable_timestamp <
```

Comment:
It's not compliant with coding standards to have nested `if` statements without need - these should be folded together.
On the other hand the change looks wrong to me. I'd have expected the original assignment to happen as well. i.e this would become:

```
         if (first_ts_upd != NULL) {                                     
                        WT_ASSERT(session,                                      
                            first_ts_upd->start_ts <= first_ts_upd->durable_ts);
                        if (r->unstable_durable_timestamp <                     
                            first_ts_upd->durable_ts)                           
                                r->unstable_durable_timestamp =                 
                                    first_ts_upd->durable_ts;                   
                        if (r->unstable_timestamp < first_ts_upd->start_ts)     
                                r->unstable_timestamp = first_ts_upd->start_ts; 
                }
```